src/app/core/admin/feedback/admin-list-feedback.component.ts

```diff
@@ -11,22 +11,57 @@ import {
 import { SystemAlertService } from '../../../common/system-alert.module';
 
 import { untilDestroyed, UntilDestroy } from '@ngneat/until-destroy';
+import cloneDeep from 'lodash/cloneDeep';
 import { Observable } from 'rxjs';
+import { take } from 'rxjs/operators';
 import { ExportConfigService } from '../../export-config.service';
+import { Feedback, FeedbackStatusOption } from '../../feedback/feedback.model';
 import { FeedbackService } from '../../feedback/feedback.service';
+import { AdminUsersService } from '../user-management/admin-users.service';
 
 @UntilDestroy()
 @Component({
-	templateUrl: 'admin-list-feedback.component.html'
+	templateUrl: 'admin-list-feedback.component.html',
+	styleUrls: ['admin-list-feedback.component.scss']
 })
 export class AdminListFeedbackComponent extends AbstractPageableDataComponent<any>
 	implements OnInit {
+	// Columns to show/hide in user table
+	columns: any = {
```

Comment:
@sshah-asymmetrik can we get better typing on this? Improved over `any` ?

Reply:
Added in our `ColumnConfig` type